gammapy/irf/rad_max.py

```diff
@@ -0,0 +1,8 @@
+from .core import IRF
+
+
+class RadMax2D(IRF):
+    """small class to exploit the IRF class to directly read in the FITS file the
+    RAD_MAX hud, this has identical axes to the EFFECTIVE AREA HDU"""
```

Comment:
Please explicitly put the expected arguments in the docstring as well as a link to the gadf specifications as done in the `EffectiveAreaTable`.